docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+db:
+  image: postgres:9.4
+  ports:
+   - "5432"
+web:
+  build: .
```

Comment:
if you have the `thoughtbot/stack-app` image in docker hub you can change this line to `image: thoughtbot/carnival` ... then you can add a `update-image` script that will do the updating of dependencies and pushing of the new image. The bonuses here is that when someone pulls this to work on it, they won't have to reinstall the dependencies.